debian_install.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+sudo apt-get install figlet
+
+echo "Nephos" | figlet
```

Comment:
Figlet is not in Debian or Ubuntu Based operating systems

Reply:
so why install it just to have it say it? Can't you generate yourself and copy paste it